libcodechecker/libhandlers/checkers.py

```diff
@@ -73,6 +73,19 @@ def add_arguments_to_parser(parser):
                         help="Show details about the checker, such as "
                              "description, if available.")
 
+    parser.add_argument('--list-profiles',
```

Comment:
maybe we do not need --list-profiles.

If the user add --profile without any profile name we list the available profiles.
Please also update the help accordingly.